django_postmark_utils/admin.py

```diff
@@ -1,3 +1,271 @@
-from django.contrib import admin
+from email.utils import formatdate
+import logging
+import pickle
+import traceback
 
-# Register your models here.
+from django.conf import settings
+from django.contrib import admin, messages
+from django.core.mail import get_connection
+from django.core.mail.message import make_msgid
+from django.core.mail.utils import DNS_NAME
```

Comment:
Some unused imports here. I'd suggest running flake8 and isort to catch other problems.